cmd/cosign/cli/generate_key_pair.go

```diff
@@ -82,6 +85,31 @@ func GenerateKeyPairCmd(ctx context.Context, kmsVal string) error {
 		}
 		fmt.Fprintln(os.Stderr, "Public key written to cosign.pub")
 		return nil
+	} else if ykVal {
+		yk, err := cosign.GetYubikey()
+		if err != nil {
+			return err
```

Comment:
I know these are nice and simple but we already have github.com/pkg/errors imported, so it would be beneficial to wrap these with more meaningful messages (i.e. what we were trying to do). That sets the tone for the rest of the codebase.